spanner/tests/system/test_system.py

```diff
@@ -282,6 +282,20 @@ def test_create_database(self):
             for database in Config.INSTANCE.list_databases()]
         self.assertIn(temp_db_id, database_ids)
 
+    def test_table_not_found(self):
+        table = 'yTable'
+        temp_db_id = 'temp_db' + unique_resource_id('_')        
+        temp_db = Config.INSTANCE.database(temp_db_id, ddl_statements=[
+            'CREATE TABLE MyTable ('
+            '    Id      STRING(36) NOT NULL,'
+            '    Field1  STRING(36) NOT NULL'
+            ') PRIMARY KEY (Id)',
+            'CREATE INDEX IDX ON {table} (Field1)'.format(table)
```

Comment:
This is invalid:

```python
>>> table = 'yTable'
>>> 'CREATE INDEX IDX ON {table} (Field1)'.format(table)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
KeyError: 'table'
```

Seems maybe you didn't have a chance to run this code?

Reply:
that is weird, I really don't know how that (Field1) got in there...

Reply:
I actually did run the system test locally, but somehow I guess I accidentally changed the code...

Reply:
never mind.  wrong error.